PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -115,6 +126,9 @@
         tkRelIso = Var("isolationR03().sumPt/tunePMuonBestTrack().pt",float,doc="Tracker-based relative isolation dR=0.3 for highPt, trkIso/tunePpt",precision=6),
         miniPFRelIso_chg = Var("userFloat('miniIsoChg')/pt",float,doc="mini PF relative isolation, charged component"),
         miniPFRelIso_all = Var("userFloat('miniIsoAll')/pt",float,doc="mini PF relative isolation, total (with scaled rho*EA PU corrections)"),
+        pt_FSR = Var("userFloat('ptFSR')",float,doc="pt of the FSR photon associated to the muon"),                 
+        eta_FSR = Var("userFloat('etaFSR')",float,doc="eta of the FSR photon associated to the muon"),                 
```

Comment:
the precision of these variables should at least be limited to that used for the nominal ones (see PhysicsTools/NanoAOD/python/common_cff.py), plus: couldn't we save space by storing pt_FSR/pt-1 with a limited precision, rather than just pt_FSR?